Publish action fails on rm of old packages

**Describe the bug**
The publish action fails trying to remove old versions when the folder is empty.
```
Run rm src/Validations/bin/Release/Triplex.Validations*.nupkg
rm: cannot remove 'src/Validations/bin/Release/Triplex.Validations*.nupkg': No such file or directory
Error: Process completed with exit code 1.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create a release, with a new tag, to trigger the action.
2. Wait for the puglish workflow to run "Publish in Nuget.org V3 Package Registry" (current name).
3. Wait for the error on the rm step (Clean old Packages)

**Expected behavior**
The rm command should be forced to avoid false negatives.

--- a/.github/workflows/nuget_push.yml
+++ b/.github/workflows/nuget_push.yml
@@ -25,7 +25,7 @@ jobs:
       run: dotnet clean --framework net6.0 --configuration Release
 
     - name: Clean old Packages
-      run: rm src/Validations/bin/Release/Triplex.Validations*.nupkg
+      run: rm --force src/Validations/bin/Release/Triplex.Validations*.nupkg
 
     - name: Restore dependencies
       run: dotnet restore

--- a/src/Validations/Validations.csproj
+++ b/src/Validations/Validations.csproj
@@ -23,7 +23,7 @@
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
         <PackageTags>Preconditions;Postconditions;Invariants;DDD;Domain Driven Design</PackageTags>
         <PackageIcon>icon.png</PackageIcon>
-        <Version>3.0.1-alpha</Version>
+        <Version>3.0.2-alpha</Version>
         <Nullable>enable</Nullable>
         <EnableNETAnalyzers>true</EnableNETAnalyzers>
         <AnalysisLevel>latest</AnalysisLevel>
@@ -36,6 +36,9 @@ Breaking Changes
     - string NotNullOrEmpty(in string?, in string)
     - string NotNullOrEmpty(in string?, in string, in string)
     - void CompliesWith(in bool, in string, in string)
+
+Bug fixes:
+  - #22 Publish action failing to remove local files before publishing. 
 
 Features
   - Introduce implicit null check for NotNullEmptyOrWhiteSpaceOnly (now NotEmptyOrWhiteSpaceOnly) and NotNullOrEmpty (now NotEmpty).

